build.proj

```diff
@@ -62,7 +63,20 @@
     <MSBuild Projects="@(PkgProject)"
              Targets="Pack" />
   </Target>
-
+  
+  <ItemGroup>
+  <TestFile Include="$(MSBuildThisFileDirectory)/test/data/external/winequality-white.csv" Url="https://archive.ics.uci.edu/ml/machine-learning-databases/wine-quality/winequality-white.csv">
+         <Result>$(MSBuildThisFileDirectory)test/data/external/winequality-white.csv</Result>
+     </TestFile>
+  </ItemGroup>
+  
+  <Target Name="DownloadExternalTestFiles" Inputs="@(TestFile)" Outputs="%(TestFile.Result)">
```

Comment:
Can you do me a favor and test this with 2 or 3 datasets downloaded from the internet?  (I guess it can be any file, it doesn't have to be a .csv file)  I just want to ensure this works with more than 1 file, and it works correctly when doing it a 2nd time that the data sets aren't downloaded on subsequent builds.

That way the next person who needs to add a dataset isn't bit by a bug here.